src/main/java/net/snowflake/ingest/streaming/internal/FlushService.java

```diff
@@ -572,7 +572,7 @@ BlobMetadata upload(String filePath, byte[] blob, List<ChunkMetadata> metadata)
           metadata.stream().mapToLong(i -> i.getEpInfo().getRowCount()).sum());
     }
 
-    logger.logDebug("Finish uploading file={}", filePath);
+    logger.logInfo("Finish uploading file={}", filePath);
```

Comment:
if we are doing this logInfo, should we include timestamp diff between start and finish?